medical_base_us/models/medical_abstract_dea.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api
+import re
```

Comment:
stdlib packages should be imported above odoo packages. Import ordering - https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports